validation-test/stdlib/AnyHashable.swift.gyb

```diff
@@ -870,6 +870,15 @@ AnyHashableTests.test("AnyHashable(NSError).base") {
   let ah = AnyHashable(NSError(domain: "Foo", code: 0))
   expectTrue(type(of: ah.base) is NSError.Type)
 }
+
+if #available(OSX 10.12, iOS 10.0, tvOS 10.0, watchOS 3.0, *) {
+  AnyHashableTests.test("Measurement.base") {
+    let measurement = NSMeasurement(doubleValue: 100,
+                                    unit: UnitLength.kilometers)
+    let ah = AnyHashable(measurement)
+    expectTrue(type(of: ah.base) is Measurement.Type)
+  }
```

Comment:
Please move the tests to `test/stdlib/TestMeasurement.swift` instead.  Please follow the pattern in `test_AnyHashableCreatedFromNSCalendar` from `test/stdlib/TestCalendar.swift`.